src/pages/html/elements/img-tag/index.md

```diff
@@ -1,15 +1,35 @@
 ---
-title: Img Tag
+title: Images
 ---
-## Img Tag
+## Images
```

Comment:
Can you change the titles back to "Img Tag"?

Reply:
I just did... is it on the Pull Request? Sorry, I'm using Git, but I'm just learning how.
Thanks!